src/Features/CSharp/Portable/CodeRefactorings/NodeSelectionHelpers.cs

```diff
@@ -18,8 +18,8 @@ internal static async Task<SyntaxNode> GetSelectedDeclarationOrVariableAsync(Cod
             // MemberDeclaration: member that can be declared in type (those are the ones we can pull up) 
             // VariableDeclaratorSyntax: for fields the MemberDeclaration can actually represent multiple declarations, e.g. `int a = 0, b = 1;`.
             // ..Since the user might want to select & pull up only one of them (e.g. `int a = 0, [|b = 1|];` we also look for closest VariableDeclaratorSyntax.
-            return await context.TryGetRelevantNodeAsync<MemberDeclarationSyntax>().ConfigureAwait(false) as SyntaxNode ??
-                await context.TryGetRelevantNodeAsync<VariableDeclaratorSyntax>().ConfigureAwait(false);
+            return await context.TryGetRelevantNodeAsync<VariableDeclaratorSyntax>().ConfigureAwait(false) as SyntaxNode ??
```

Comment:
Switch the order here because 
```
class Bar
{
 pu$$blic int i;
}
```
In such case, helper here would return FieldDeclarationNode first, but what we want is VariableDeclarator in the field

Reply:
Make sure to update the comment to indicate that the ordering is important

Reply:
Updated the comments